pom.xml

```diff
@@ -73,6 +73,7 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>logging-interceptor</artifactId>
+      <version>5.0.0-alpha.11</version>
```

Comment:
should not be a pre-release version and is managed by the bom - so pickup the version from there.
```suggestion
```